solarforecastarbiter/reports/figures/plotly_figures.py

```diff
@@ -359,7 +384,94 @@ def _plot_fx_timeseries(fig, timeseries_value_df, timeseries_meta_df, axis):
             **plot_kwargs)
         # collect in list
         gos.append((metadata['pair_index'], go_))
-    # Add traces in order of pair index
+
+    palette = cycle(PROBABILISTIC_PALETTES)
+    for dist_hash in np.unique(distribution_meta['distribution']):
+        # indices to constant values in the metadata df
+        cv_indices = timeseries_meta_df['distribution'] == dist_hash
+
+        # sort constant values
+        cv_metadata = timeseries_meta_df[cv_indices]
+        cv_metadata = cv_metadata.sort_values('constant_value')
+        cv_metadata = cv_metadata.reset_index()
+
+        # Get a colormap for mapping fill colors
+        color_map = cm.get_cmap(next(palette))
+        color_scaler = cm.ScalarMappable(
+            Normalize(vmin=0, vmax=1),
+            color_map,
+        )
+
+        def _get_fill_color(percentile):
+            normalized_value = percentile / 100
+            return rgb2hex(color_scaler.to_rgba(normalized_value))
+
+        symmetric_percentiles = _percentiles_are_symmetric(cv_metadata)
+        # Plot confidence intervals
+        for idx, cv in cv_metadata.iterrows():
+            pair_idcs = timeseries_value_df['pair_index'] == cv['pair_index']
+            data = _fill_timeseries(
+                timeseries_value_df[pair_idcs],
+                cv['interval_length'])
+
+            # Fill missing data with 0 to avoid plotly bugs encountered with
+            # go.Scatter fill and missing data.
+            data = data.fillna(0)
+
+            if idx == 0:
+                # The first value will act as the lower bound for other values
+                # to fill down to.
+                fill = None
+                showlegend = True
+            else:
+                fill = 'tonexty'
+                showlegend = False
+
+            # Split name of the distribution from the current constant value
+            constant_label_index = cv['forecast_name'].find('Prob(')-1
```

Comment:
```suggestion
            constant_label_index = cv['forecast_name'].find('Prob(') - 1
```

Reply:
this will break if we change the naming pattern. The only way I can see to improve it is to add yet more metadata to the dataframe. So ok to keep it this way for now.